### PR TITLE
feat(tokens): update primitive json

### PR DESCRIPTION
**Pull request by BenOsodrac**
Issue number: internal

---------

## What is the new behavior?

- Update primitive json tokens
- Update generated files on foundations folder.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other Information
[Tokens Preview Sample](https://ionic-framework-fugbxhshx-ionic1.vercel.app/src/foundations/tokens.preview.html)

### CODE REVIEW

File: core/src/foundations/ionic.root.scss [hunk 1]
```diff
@@ -59,6 +59,7 @@
   --ionic-space-600: 24px;
   --ionic-space-700: 28px;
   --ionic-space-800: 32px;
+  --ionic-space-850: 34px;
```
Comment (brandyscarney) [inline core/src/foundations/ionic.root.scss:62]: Is there a reason we are adding 850 but not 750, 650, etc? Are we just adding tokens as we need them?

Comment (brandyscarney) [reply core/src/foundations/ionic.root.scss:62]: I spoke with Beth and she said we can review all of the tokens once they are implemented. 👍

File: core/src/foundations/tokens.preview.html [hunk 2]
```diff
@@ -1725,6 +1727,10 @@ <h1>Space Tokens</h1>
             <div class="space-token" style="margin: 32px">-space-800 (32px)</div>
           </div>
 
+          <div class="spacing-wrapper">
+            <div class="space-token" style="margin: 34px">-space-850 (34px)</div>
```
Comment (brandyscarney) [inline core/src/foundations/tokens.preview.html:1731]: Why don't these use the classes like `.ionic-margin-space-850`?

**Review by brandyscarney — APPROVED** [review]

File: core/src/foundations/tokens.preview.html (hunk 2 above)
Comment (BenOsodrac) [reply core/src/foundations/tokens.preview.html:1731]: At the time I wanted to have this file abstract of the other ones, to be as `pure` as possible. 
But it makes sense, I think we can do that in a future improvement to this 👍

Comment (brandyscarney) [reply core/src/foundations/tokens.preview.html:1731]: That's fine with me!

**Pull request merged**